frontend/nuxt.config.ts

```diff
@@ -107,6 +137,7 @@ export default defineNuxtConfig({
     },
   },
   site: {
+    indexable: isProdNotPlaywright,
```

Comment:
I _think_ this is right? We want `indexable` to be true on production but not on staging.